critiquebrainz/ws/review/views.py

```diff
@@ -350,7 +350,10 @@ def review_list_handler():
 
     user_id = Parser.uuid('uri', 'user_id', optional=True)
     # TODO: "rating" sort value is deprecated and needs to be removed.
-    sort = Parser.string('uri', 'sort', valid_values=['popularity', 'published_on', 'rating'], optional=True)
+    sort = Parser.string('uri', 'sort', valid_values=['popularity', 'published_on', 'rating', 'created'], optional=True)
```

Comment:
I had a quick look in critiquebrainz/ws/review/views_test.py and don't see any tests that check these parameters (or if they're rewritten correctly). Do you think that they are necessary?

Reply:
I'll add the tests. In general, I feel CB is missing a lot of tests. Many of the bug fixes we did could have been easily caught by simple tests and fixed before merging. Moving forward I think we should add tests for at least the changes we do. Later, we can try to add tests for features that already present in the code base but do not have tests.

Reply:
This is a great way of working. When fixing something, leave the code in a better state than when you arrived. This could include cleaning up some comments, fixing duplicated code, doing something in a way that is easier to understand, or writing tests. Keep this in mind especially when you're fixing these simple tickets.